ui/components/app/token-cell/token-cell.js

```diff
@@ -19,39 +22,57 @@ export default function TokenCell({
 }) {
   const userAddress = useSelector(getSelectedAddress);
   const t = useI18nContext();
-
+  const tokenList = useSelector(getTokenList);
+  const tokenData = Object.values(tokenList).find(
+    (token) => token.symbol === symbol,
+  );
+  const title = tokenData?.name || symbol;
   const formattedFiat = useTokenFiatAmount(address, string, symbol);
   const warning = balanceError ? (
-    <span>
+    <Text as="span">
       {t('troubleTokenBalances')}
-      <a
+      <ButtonLink
         href={`https://ethplorer.io/address/${userAddress}`}
-        rel="noopener noreferrer"
-        target="_blank"
+        externalLink
         onClick={(event) => event.stopPropagation()}
-        style={{ color: 'var(--color-warning-default)' }}
+        textProps={{
+          color: TextColor.warningDefault,
+        }}
       >
         {t('here')}
-      </a>
-    </span>
+      </ButtonLink>
+    </Text>
   ) : null;
 
   return (
-    <AssetListItem
-      className={classnames('token-cell', {
-        'token-cell--outdated': Boolean(balanceError),
-      })}
-      iconClassName="token-cell__icon"
-      onClick={onClick.bind(null, address)}
-      tokenAddress={address}
-      tokenSymbol={symbol}
-      tokenDecimals={decimals}
-      tokenImage={image}
-      warning={warning}
-      primary={`${string || 0}`}
-      secondary={formattedFiat}
-      isERC721={isERC721}
-    />
+    <>
+      {process.env.MULTICHAIN ? (
+        <MultichainTokenListItem
+          onClick={() => onClick(address)}
+          tokenSymbol={symbol}
+          tokenImage={image}
+          primary={`${string || 0}`}
+          secondary={formattedFiat}
+          title={title}
+        />
+      ) : (
+        <AssetListItem
+          className={classnames('token-cell', {
+            'token-cell--outdated': Boolean(balanceError),
+          })}
+          iconClassName="token-cell__icon"
+          onClick={() => onClick(address)}
+          tokenAddress={address}
+          tokenSymbol={symbol}
+          tokenDecimals={decimals}
+          tokenImage={image}
+          warning={warning}
+          primary={`${string || 0}`}
+          secondary={formattedFiat}
+          isERC721={isERC721}
+        />
+      )}
+    </>
   );
 }
 
```

Comment:
Would also be good to get a story for this file

Reply:
Currently we have native token and ERC20 tokens. In this `token cell `we are just passing those ERC20 tokens to MultichainTokenListItem and AssetList. So, I am not sure if we should add a separate story here.